ImperatorToCK3/CK3/Titles/TitleHistory.cs

```diff
@@ -5,32 +5,31 @@ namespace ImperatorToCK3.CK3.Titles {
 	public class TitleHistory {
 		public TitleHistory() { }
 		public TitleHistory(History history, Date ck3BookmarkDate) {
-			this.history = history;
-			var holderFromHistory = history.GetSimpleFieldValue("holder", ck3BookmarkDate);
-			if (holderFromHistory is null) {
-				Logger.Warn("TitleHistory: holder should not be null!");
-			} else {
-				Holder = holderFromHistory;
-			}
+			History = history;
 			Liege = history.GetSimpleFieldValue("liege", ck3BookmarkDate);
-			Government = history.GetSimpleFieldValue("government", ck3BookmarkDate);
 
 			var developmentLevelOpt = history.GetSimpleFieldValue("development_level", ck3BookmarkDate);
 			if (developmentLevelOpt is not null) {
 				DevelopmentLevel = int.Parse(developmentLevelOpt);
 			}
 		}
 		public void Update(HistoryFactory historyFactory, BufferedReader reader) {
-			historyFactory.UpdateHistory(history, reader);
+			historyFactory.UpdateHistory(History, reader);
 		}
 
-		// These values are open to ease management.
-		// This is a storage container for CK3::Title.
-		public string Holder { get; set; } = "0"; // ID of Character holding the Title
+		public string GetHolderId(Date date) {
+			var idFromHistory = History.GetSimpleFieldValue("holder", date);
+			if (idFromHistory is not null) {
+				return idFromHistory;
+			}
+			return "0";
+		}
+		public string? GetGovernment(Date date) {
+			return History.GetSimpleFieldValue("government", date);
+		}
 		public string? Liege { get; set; }
-		public string? Government { get; set; }
 		public int? DevelopmentLevel { get; set; }
 
-		private readonly History history = new();
+		public History History { get; } = new();
```

Comment:
The type and the variable name are exactly the same? Won't that cause confusion?

Reply:
Renamed to InternalHistory.